### PR TITLE
File crawling for plans

### PR DESCRIPTION
**Pull request by GalGend**
This PR is only for crawlign and saving mavat files

### CODE REVIEW

File: server/api/lib/mavat/files/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    if (file.file_icon.includes('DWG')) return FILE_TYPES.DWG;
+    return FILE_TYPES.UNKNOWN;
+}
+
+const isKML3d = ( { kind, name } ) => {
+	return kind=== 'קבצים דיגיטליים' && name.includes('KML') && name.includes('תלת מימד');
+};
+
+const getFileExtension = ( file ) => {
+    if (isKML3d(file)) return 'kml';
```
Comment (GalGend) [inline server/api/lib/mavat/files/index.js:20]: Here I should change to use the enum

File: server/api/lib/mavat/index.js [hunk 2]
```diff
@@ -349,6 +368,7 @@ const getByPlan = plan =>
 	init()
 		.then(() => {
 			return plan.get('plan_url') ? fetch(plan.get('plan_url')) : search(plan.get('PL_NUMBER'));
+			//return fetch(plan.plan_url);
```
Comment (florpor) [inline server/api/lib/mavat/index.js:371]: remove comment

File: server/api/lib/mavat/files/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+const { File } = require('../../../model');
```
Comment (florpor) [inline server/api/lib/mavat/files/index.js:1]: file model is not exported at `api/model/index.js`. either do that or change this require

**Review by florpor — CHANGES_REQUESTED** [review]
see comments above, and also please add a basic test for this (can be part of the existing crawler tests) so we know it works (at least basically) through package upgrades and code changes.

**Review by florpor — APPROVED** [review]

**Pull request merged**